snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/SnomedDescriptionCreateRequest.java

```diff
@@ -145,9 +153,17 @@ public String execute(TransactionContext context) {
 			acceptabilityUpdate.setDescriptionId(description.getId());
 			acceptabilityUpdate.execute(context);
 			
+			final SnomedInactivationReasonUpdateRequest inactivationUpdate =  new SnomedInactivationReasonUpdateRequest<>(
```

Comment:
This should be executed when the `inactivationIndicator` is not `null`.

Reply:
After a short discussion about this change, I've changed my mind, let's leave it like this for now.